apps/dashboard/src/main/java/com/akto/action/DashboardAction.java

```diff
@@ -72,39 +77,92 @@ public String fetchRiskScoreCountMap(){
 
     Set<Integer> deactivatedCollections = UsageMetricCalculator.getDeactivated();
 
-    private long totalIssuesCount = 0;
-    private long oldOpenCount = 0;
+    List<Integer> totalIssuesCountDayWise;
     public String findTotalIssues() {
-        Set<Integer> demoCollections = new HashSet<>();
-        demoCollections.addAll(deactivatedCollections);
-        demoCollections.add(RuntimeListener.LLM_API_COLLECTION_ID);
-        demoCollections.add(RuntimeListener.VULNERABLE_API_COLLECTION_ID);
+        long daysBetween = (endTimeStamp - startTimeStamp) / ONE_DAY_TIMESTAMP;
```

Comment:
In case startTimestamp is 0,  according to the query below, we will fetch all the data from the start till now. That can be a lot of data. Is this expected ?